src/components/ImageExportDialog.tsx

```diff
@@ -52,15 +56,37 @@ export type ExportCB = (
   scale?: number,
 ) => void;
 
-const ExportModal = ({
+const ExportButton: React.FC<{
+  color: keyof OpenColor;
+  onClick: () => void;
+  title: string;
+  shade?: number;
+}> = ({ children, title, onClick, color, shade = 6 }) => {
+  return (
+    <button
+      className="ExportDialog-imageExportButton"
```

Comment:
we should try to follow consistency when adding classnames in general, rn class names start with upper & lower both but we should follow one, since we scope with `excalidraw` hence I would suggest we always name classes  starting with lower case.

and similarly as mentioned above if we can add the css vars to the above as its a unique class for the component

Reply:
The styleguide is generally as such: component classes are capitalized, utility classes and other classes are lower-cased.

As for whether we want to prefix all our classes with excalidraw: I'd leave that for a separate PR as that is a major change across the whole codebase and unrelated to this redesign.

Reply:
Yes I am talking in general and not specific to this PR so we can do it in a separate PR.